### PR TITLE
Add more marin splash text

### PR DESCRIPTION
**Pull request by muffinjets**
There's obviously that really long one, I think it could work well in a race if it's the only one of that length.

### CODE REVIEW

File: patches/aesthetics.py [hunk 1]
```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
         "#####, how do I download RAM?",
         "Is this a delayed April Fool's Joke?",
         "Dayoman is close to getting 120% in 1:20!",
-        "Play as if your footage will go in a\nSummoning Salt video."
+        "Play as if your footage will go in a\nSummoning Salt video.",
+        "I hope you prepared for our date later.",
+        "Isn't this the game where you date a seagull?",
+        "!tele broken, pls fix",
+        "You look pretty good for a guy who probably drowned.",
+        "I heard Matty's a cheater.",
```
Comment (daid) [inline patches/aesthetics.py:269]: People not aware of what cheating means for insiders might get the wrong idea.

File: patches/aesthetics.py [hunk 2]
```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
...
+        "Batman Contingency Plan: Link.  Step 1: Disguise yourself as a maiden to attract the young hero.",
+        "I have flooded the Wind Fish's Egg with a deadly neurotoxin.",
+        "Ahh, General #####.",
+        "Finally, Link's Awakening!",
+        "Is the Wind Fish dreaming that he's sleeping in an egg?  Or is he dreaming that he's you?",
+        "Save Koholint.  By destroying it.  Huh?  Don't ask me, I'm just a kid!",
+        "There aren't enough women in this village to sustain a civilization.",
+        "So does this game take place before or after Oracles?",
+        "Have you tried the critically acclaimed MMORPG FINAL FANTASY XIV that has a free trial up to level 60 including the Heavensward expansion?",
+        "The thumbs-up sign had been used by the Galactic Federation for ages. Me, I was known for giving the thumbs-down during briefing. I had my reasons, though... Commander Adam Malkovich was normally cool and not one to joke around, but he would end all of his mission briefings by saying, 'Any objections, Lady?'",
```
Comment (daid) [inline patches/aesthetics.py:305]: This might give issues with text table space.

File: patches/aesthetics.py [hunk 3]
```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
...
+        "No eggs were harmed in the making of this game.",
+        "I'm helping the goat, \ncatfishing Mr. Write is kinda the goal.",
+        "There are actually two LADX randomizers.",
+        "Did you actually dump this cart?"
+        "You're not gonna cheat... \n ...right?",
+        "Mamu's singing is so bad it wakes the dead.",
+        "Don't forget the Richard picture.",
+        "Are you sure you wanna do this?  I kinda like this island.",
+        "SJ, BT, WW, OoB, HIJKLMNOP.",
+        "$5 in the swear jar.  Now.",
```
Comment (daid) [inline patches/aesthetics.py:284]: Is there a symbol for `$`?

Comment (muffinjets) [reply patches/aesthetics.py:284]: Good point, I'll edit it when I get back home

File: patches/aesthetics.py (hunk 1 above)
Comment (muffinjets) [reply patches/aesthetics.py:269]: Asking Matty about it now.

File: patches/aesthetics.py (hunk 2 above)
Comment (muffinjets) [reply patches/aesthetics.py:305]: I'll give it a shot, see if it runs into any issues.

File: patches/aesthetics.py [hunk 4]
```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
...
+        "Play as if your footage will go in a\nSummoning Salt video.",
+        "I hope you prepared for our date later.",
+        "Isn't this the game where you date a seagull?",
+        "!tele broken, pls fix",
+        "You look pretty good for a guy who probably drowned.",
+        "I heard Matty's a cheater.",
+        "Remember, we race on Sundays.",
+        "Pink Batman delayed this race for lunch.",
+        "This randomizer was made possible by players like you. \n \n Thank you!",
+        "Now with real fake doors!",
```
Comment (kabarakh) [inline patches/aesthetics.py:273]: I think this was supposed to be "floors" ("real fake floors" is a thing Matty says when the Eagle quick kill is possible)

File: patches/aesthetics.py (hunk 1 above)
Comment (muffinjets) [reply patches/aesthetics.py:269]: I understand and agree.

File: patches/aesthetics.py (hunk 3 above)
Comment (muffinjets) [reply patches/aesthetics.py]: Adjusted a few lines that had symbols that probably aren't in the LADX font.

File: patches/aesthetics.py (hunk 2 above)
Comment (muffinjets) [reply patches/aesthetics.py:305]: I genned a seed with this text, I didn't run into issues with generation

Comment (muffinjets) [reply patches/aesthetics.py:304]: I don't know what kinds of issues to look out for, So I wouldn't know how to bug-hunt on this one, sorry

**Pull request merged**